android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -25,10 +26,38 @@ class JsonConverter {
          */
         @JvmStatic
         fun toPlayerConfig(json: ReadableMap?): PlayerConfig {
+            var config = PlayerConfig()
             if (json != null && json.hasKey("licenseKey")) {
-                return PlayerConfig(key = json.getString("licenseKey"))
+                config = PlayerConfig(key = json.getString("licenseKey"))
             }
-            return PlayerConfig()
+            if (json != null && json.hasKey("playbackConfig")) {
+                var playbackConfigJson = json.getMap("playbackConfig")
+                /**
+                 * Specifies whether autoplay is enabled.
+                 *
+                 * Default is `false`.
+                 */
+                if(playbackConfigJson != null && playbackConfigJson.hasKey("isAutoplayEnabled")) {
```

Comment:
NIT (for all occurrences):
```suggestion
                if (playbackConfigJson?.hasKey("isAutoplayEnabled") == true) {
```